internal/alert/manager.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	RoutingCfgPath          string
 	PagerdutyAlertEventsURL string
 	RoutingParams           *core.AlertRoutingParams
+	SNSConfig               *client.SNSConfig
```

Comment:
Can we add a comment detailing why this alerting resource isn't part of the `AlertRoutingParams`?

Reply:
[74f2414](https://github.com/base-org/pessimism/pull/200/commits/74f241413764e3b9caf17a604ffe27f2bc28480c)